dev/isaccml.env.default

```diff
@@ -0,0 +1,9 @@
+# Example docker-compose environment file
+# Copy to isaccml.env and modify as necessary
+# https://docs.docker.com/compose/env-file/
+
+# Variables defined in this file will only be available to containers/images
+# ie not for interpolation in docker-compose YAML files
+SECRET_KEY=
+
+TORCH_MODEL_PATH=
```

Comment:
Since there's already a default in `dev/docker-compose.yaml`, I'd comment this out to make it more obvious that it is no longer required

```suggestion
# TORCH_MODEL_PATH=
```